lib/logflare_grpc/endpoint.ex

```diff
@@ -0,0 +1,5 @@
+defmodule LogflareGrpc.Endpoint do
+  use GRPC.Endpoint
+
+  run(LogflareGrpc.Trace.Server, interceptors: [{GRPC.Logger.Server, level: :info}])
```

Comment:
log level should match the log level env var. looks like this is a macro, so the value might not match the runtime log level config.

Reply:
Will check what I can do

Reply:
Yeah, during compilation there's no way but there will be a release later of grpc with telemetry and then we can plug into that for this type of information.

Honestly, this was only used for debugging, it doesn't offer much now.